frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -73,27 +75,29 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
       case InferenceServiceModelState.UNKNOWN:
         return (
           <Icon
-            data-testid="status-tooltip"
+            data-testid="status-popover"
             role="button"
             aria-label="warning icon"
             status="warning"
             isInline
             tabIndex={0}
             iconSize={iconSize}
+            style={{ cursor: 'pointer' }}
           >
             <OutlinedQuestionCircleIcon />
           </Icon>
         );
       default:
         return (
           <Icon
-            data-testid="status-tooltip"
+            data-testid="status-popover"
             role="button"
             aria-label="warning icon"
             status="warning"
             isInline
             tabIndex={0}
             iconSize={iconSize}
+            style={{ cursor: 'pointer' }}
```

Comment:
You should use a PF `<Button variant="link" isInline icon={...} />` as a child of the `Popover` to create a semantic element that can gain focus and has the correct cursor by default.